apollo-federation/src/error/mod.rs

```diff
@@ -126,6 +126,11 @@ pub enum SingleFederationError {
     // This is a known bug that will take time to fix, and does not require reporting.
     #[error("{message}")]
     InternalUnmergeableFields { message: String },
+    #[error("[{subgraph}] {error}")]
+    SubgraphError {
```

Comment:
When we split QP and composition error variants, is this variant only for composition?

Reply:
Yep, this is only for composition.